manifests/t/the-sz/Royal/1.67/the-sz.Royal.yaml

```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
+
+PackageIdentifier: the-sz.Royal
+PackageVersion: 1.67

```

Comment:
Package has no version in the ARP entry, this will lead to upgrade loop
![image](https://user-images.githubusercontent.com/12611259/125691950-ce87b657-c9d9-4c18-872e-2667795beada.png)

Reply:
fixed with new commit